modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/extra/remote-app/src/routes/reports/generateReport/index.tsx

```diff
@@ -158,38 +146,31 @@ const GenerateReport = () => {
 					<div className="col">
 						<Form.Input
 							{...formProps}
-							label="Initial Company Name"
-							name="initialCompanyName"
+							label="Company Name"
+							name="organizationName"
 							placeholder="Company Name"
 						/>
 					</div>
-
-					<div className="col">
-						<Form.Input
-							{...formProps}
-							label="Final Company Name"
-							name="finalCompanyName"
-							placeholder="Initial Company Name"
-						/>
-					</div>
 				</div>
 
 				<div className="row">
 					<div className="col">
 						<label>Statuses</label>
 
-						{statuses.map((status: any, index: string) => (
+						{statuses.map((status: Statuses, index: string) => (
```

Comment:
Why is index being typed as a string?
```suggestion
						{statuses.map((status: Statuses, index: string ???) => (
```